modules/services/gpg-agent.nix

```diff
@@ -41,7 +41,7 @@ in
 
       home.sessionVariables =
         optionalAttrs cfg.enableSshSupport {
-          SSH_AUTH_SOCK = "\${XDG_RUNTIME_DIR}/gnupg/S.gpg-agent.ssh";
+          SSH_AUTH_SOCK = "\$XDG_RUNTIME_DIR/gnupg/S.gpg-agent.ssh";
```

Comment:
Because curly braces are not portable to fish. I don't know what the best portable solution that includes PAM would be.